omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestController1_10Test.java

```diff
@@ -439,6 +439,27 @@ public void postAttachment_shouldNotUploadFileAboveSizeLimit() throws Exception
 		SimpleObject response = deserialize(handle(request));
 	}
 	
+	@Test(expected = IllegalRequestException.class)
+	public void postAttachment_shouldNotUplodExecutbleFile() throws Exception {
```

Comment:
There's a typo here by maybe rename this to `postAttachment_shouldThrowOnExecutableFile`.